.doc_gen/readmes/readme.txt

```diff
@@ -0,0 +1,66 @@
+# {{ service_name }} code examples for the {{ short_sdk_name }}
+## Overview
+These examples show how to {{ code_examples_actions }} using the {{ short_sdk_name }}.
+
+{{ service_blurb_from_website }}
+
+## ⚠️ Important
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single actions
+Code excerpts that show you how to call individual service functions.
+{% for action in actions %}
+* {{ action }}
+{% endfor %}
+
+{% if scenarios %}
+### Scenarios
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+{% for scenario in scenarios %}
+* {{ scenario }}
+{% endfor %}
+{% endif %}
+
+{% if cross_service_examples %}
+### Cross-service examples
+Sample applications that work across multiple AWS services.
+{% for cross_service_example in cross_service_examples %}
+* {{ cross_service_example }}
+{% endfor %}
+{% endif %}
+
+## Run the examples
+
+{% if prerequisites %}
+### Prerequisites
+{% for prereq in prerequisites %}
+1. {{ prereq }}
+{% endfor %}
+{% endif %}
+
+### Instructions
+{{ code_instructions }}
+
+## Contributing
+Code examples thrive on community contribution!
+
+* To propose a new example, submit an [Enhancement Request](https://github.com/awsdocs/aws-doc-sdk-examples/issues/new?assignees=octocat&labels=type%2Fenhancement&template=enhancement.yaml&title=%5BEnhancement%5D%3A+%3CDESCRIPTIVE+TITLE+HERE%3E) (~2 min).
+* To fix a bug, submit a [Bug Report](https://github.com/awsdocs/aws-doc-sdk-examples/issues/new?assignees=octocat&labels=type%2Fbug&template=bug.yaml&title=%5BBug%5D%3A+%3CDESCRIPTIVE+TITLE+HERE%3E) (~5 min).
+* To learn more about the contributing process, see [CONTRIBUTING.md](../../../CONTRIBUTING.md)
+### Testing
+⚠️ Running these tests might result in charges to your AWS account.
+
+{{ test_instructions }}
+
+## Additional resources
+* [Service Developer Guide]({{service_developer_guide}})
+* [Service API Reference]({{service_api_reference_guide}})
+* [SDK API reference guide]({{language_sdk_reference_guide}})
```

Comment:
These are not the right links, and are intended to be specific to the service:
* Service developer guide should link to the dev/user guide for the service, such as "S3 User Guide".
* Service API reference should link to the wire API reference for the service, such as "S3 API Reference".
* SDK API reference should link to the service section of the SDK API reference, such as "Boto3 S3 client".

Reply:
+1, links and titles are service-specific.

Reply:
I'm confused by this. Can you provide an example of what you mean?